packages/flutter_tools/lib/src/test/coverage_collector.dart

```diff
@@ -66,6 +66,11 @@ class CoverageCollector extends TestWatcher {
     assert(data != null);
 
     _logMessage('($observatoryUri): collected coverage data; merging...');
+    _addHitmap(await coverage.createHitmap(
+      data['coverage'] as List<Map<String, dynamic>>,
+      packagesPath: globalPackagesPath,
+      checkIgnoredLines: true,
+    ));
     _addHitmap(await coverage.createHitmap(data['coverage'] as List<Map<String, dynamic>>));
```

Comment:
This looks like you're adding the hitmap twice here and below

Reply:
oh good catch, missed it during the re-sync